style.css

```diff
@@ -1,7 +1,356 @@
 .basis {
+  background-color: #e3eafe;
   font-family: "Roboto", serif;
   font-weight: 900;
-  height: 2766px;
-  width: 1440px;
+  margin: 0;
+  max-width: 1440px;
+  min-width: 300px;
+}
+
+@media screen and (max-width: 320px) {
+  .basis {
+    max-height: 1635px;
+    max-width: 320px;
+  }
+}
+
+.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  margin-bottom: 22px;
+  min-height: 334px;
+}
+
+@media screen and (max-width: 320px) {
+  .header {
+    background-image: url(images/nav/adapt-bgr.jpg);
+    max-height: 334px;
+    max-width: 320px;
+  }
+}
+
+.nav {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  margin-bottom: 131px;
+}
+
+.nav__link {
+  color: #fff;
+  font-size: 18px;
+  margin-top: 70px;
+  text-decoration: none;
+}
+
+.nav__link:first-child {
+  height: 21px;
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(2) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(3) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(4) {
+  margin-right: 130px;
+}
+
+.nav__link:hover {
+  color: #fff;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.burger {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  display: none;
+  height: 3px;
+  position: relative;
+  width: 23px;
+}
+
+.burger::before {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  content: " ";
+  height: 3px;
+  position: absolute;
+  top: 5px;
+  width: 23px;
+}
+
+.burger::after {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  bottom: 5px;
+  content: " ";
+  height: 3px;
+  position: absolute;
+  width: 23px;
+}
+
+@media screen and (max-width: 320px) {
+  .nav { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)